proto/tabletmanagerdata.proto

```diff
@@ -360,7 +360,6 @@ message ReplicationStatusRequest {
 
 message ReplicationStatusResponse {
   replicationdata.Status status = 1;
-  bool backup_running = 2;
```

Comment:
A field for this is already present in replicationdata.Status, no need to add another one here which just led to confusion in the previous PR.